codalab/model/worker_model.py

```diff
@@ -154,6 +157,7 @@ def get_workers(self):
             ).fetchall()
             worker_run_rows = conn.execute(cl_worker_run.select()).fetchall()
 
+        # free_disk_bytes will be set later.
```

Comment:
You mean `run_uuids`?  Make this comment on the same line as `'run_uuids': []`.

Reply:
Hmm, actually, why this will be set later?

Reply:
All I mean is that currently, it looks like from `'run_uuids': []` that it's set to empty, but a few lines later, it gets populated.

Reply:
I see.